squire_web/src/tournament/players/selected.rs

```diff
@@ -19,16 +25,85 @@ impl SelectedPlayer {
         digest
     }
 
+    pub fn update_round(&mut self, rid: Option<RoundId>) -> bool {
```

Comment:
The `update` method should probably be expanded to include this. Instead, that function should take a `SelectedPlayerMessage`, and we can define that message as:
```rust
pub enum SelectedPlayerMessage {
    PlayerSelected(PlayerId),
    InfoSelected(SelectedPlayerInfo),
}
```
This way, the interface of the `SelectedPlayer` looks identical to all the other components.